distro/runtime/concierge/smarthome.xargs

```diff
@@ -3,7 +3,7 @@
 # Version configuration
 -Dconcierge.version=5.0.0
 -Djetty.version=9.3.10
--Desh.version=0.9.0.b1
+-Desh.version=0.9.0-SNAPSHOT
```

Comment:
Ping! 0.9.0.b2 is now available, so you can include the audio/voice features when using it.